uportal-war/src/main/java/org/jasig/portal/security/IPermissionPolicy.java

```diff
@@ -18,34 +18,40 @@
  */
 package org.jasig.portal.security;
 
+import org.jasig.portal.permission.IPermissionActivity;
+import org.jasig.portal.permission.IPermissionOwner;
+import org.jasig.portal.permission.target.IPermissionTarget;
+
 /**
  * Defines a pluggable strategy for evaluating the permissions associated
  * with a principal.
  * @author Dan Ellentuck
- * @version $Revision$
  * @see org.jasig.portal.security.IAuthorizationService
  * @see org.jasig.portal.security.IPermission
  */
 public interface IPermissionPolicy {
-/**
- * Answers if the owner has authorized the principal to perform the activity 
- * on the target, based on permissions provided by the service.  Params 
- * <code>service</code>, <code>owner</code> and <code>activity</code> must 
- * be non-null.
- *
- * @return boolean
- * @param service org.jasig.portal.security.IAuthorizationService
- * @param principal org.jasig.portal.security.IAuthorizationPrincipal
- * @param owner java.lang.String
- * @param activity java.lang.String
- * @param target java.lang.String
- * @exception org.jasig.portal.AuthorizationException
- */
-public boolean doesPrincipalHavePermission
-   (IAuthorizationService service,
-    IAuthorizationPrincipal principal, 
-    String owner, 
-    String activity, 
-    String target) 
-throws org.jasig.portal.AuthorizationException;
+
+    /**
+     * Answers if the owner has authorized the principal to perform the activity 
+     * on the target, based on permissions provided by the service.  Params 
+     * <code>service</code>, <code>owner</code> and <code>activity</code> must 
+     * be non-null.
+     *
+     * @param service org.jasig.portal.security.IAuthorizationService
+     * @param principal org.jasig.portal.security.IAuthorizationPrincipal
+     * @param owner The 'namespace' of the activity
+     * @param activity The behavior that requires permission
+     * @param target The object upon which the behavior will be invoked
+     * @return TRUE if the principal has permission to perform the specified
+     * activity on the specified target
+     * @exception org.jasig.portal.AuthorizationException
```

Comment:
API method is effectively new for uPortal 4.3.  Worth an `@since` acknowledging this?